webapi/src/it/scala/org/knora/webapi/TestContainers.scala

```diff
@@ -27,6 +28,10 @@ import org.testcontainers.containers.{BindMode, GenericContainer}
   */
 object TestContainers {
 
+    // get local IP address, which we need for SIPI
+    val localhost: InetAddress = InetAddress.getLocalHost
+    val localIpAddress: String = localhost.getHostAddress
```

Comment:
This should be your machine's IP. Could you maybe run it in debug mode and see what you get here?

Reply:
I think it's a static initialiser bug: if so, these `val`s need to be `lazy` because otherwise they might not be initialised before they're needed. Will confirm after lunch.